python/cuml/common/handle.pyx

```diff
@@ -60,8 +60,8 @@ cdef class Handle:
     # 'size_t'!
     cdef size_t h
 
-    def __cinit__(self):
-        self.h = <size_t>(new cumlHandle())
+    def __cinit__(self, n_streams=3):
```

Comment:
Wouldn't we want to set this to the default? I think RF is going to be the only algorithm using this currently, perhaps KNN in the near future.

Reply:
Done.